action.yml

```diff
@@ -3,14 +3,29 @@ description: 'A GitHub Action to run a UI test based on user instructions'
 author: 'foreai-co'
 
 inputs:
+  website:
+    description: 'The instructions to test'
+    required: true
+    default: ''
+
+  test-name:
+    description: 'A unique name to identify the test'
+    required: true
+    default: ''
+
   instructions:
     description: 'The instructions to test'
     required: true
-    default: 'Hello, World!'
+    default: ''
+  
+  cj-token:
+    description: 'The token required to access the tool'
+    required: true
+    default: ''
 
 outputs:
   result:
-    description: 'The test result'
+    description: 'URL where the result will be available.'
```

Comment:
strictly speaking the script returns a message that includes a URL, not just the url

Reply:
Changed.